### ISSUE TITLE
[feature request] Add 'VAE/Text Encoder' and 'Checkpoint' in flux/xl/sd Defaults in Settings

### ISSUE THREAD

**Issue by ostap667inbox** [lllyasviel/stable-diffusion-webui-forge#2305]
When working, you often have to switch between SDXL and Flux checkpoints.
Every time you do this, you have to set the VAE for the SDXL checkpoint or the VAE/Text Encoder list for the Flux checkpoint. 

Also when switching xl/flux you have to select a checkpoint from the list every time. If you regularly work with the same settings, this is the same constantly repeating actions. It would be great to be able to set Default checkpoint and default VAE/Text encoders in Settings in 'UI defaults...' for flux/xl/sd.

**Comment by Juqowel** [comment]
Yes. It would be great, but with the latest ones instead of the default.

**Comment by ostap667inbox** [comment]
> Yes. It would be great, but with the latest ones instead of the default.

I agree. Your version is even better suited for this task.
